5.5/root/usr/bin/run-mysqld

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+export_vars=$(cgroup-limits); export $export_vars
```

Comment:
@mnagy can explainthe export vars
On Apr 27, 2016 16:02, "Vyacheslav Semushin" <notifications@github.com>
wrote:

> In 5.5/root/usr/bin/run-mysqld
> <https://github.com/openshift/mysql/pull/130#discussion_r61261955>:
>
> > @@ -1,5 +1,6 @@
> >  #!/bin/bash
> >
> > +export_vars=$(cgroup-limits); export $export_vars
>
> I don't know, I've used the same code as @mfojtik
> <https://github.com/mfojtik>. My idea was to do the same things similar
> in a different places.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/mysql/pull/130/files/9ee28581ccb466e916c9d81d9f6f9a2663e5d762#r61261955>
>